File: src/main/java/client/database/data/PlayerLifeData.java
package client.database.data;

public class PlayerLifeData {
   private int lifeId;

   private String type;

   private int cy;

   private int f;

   private int fh;

   private int rx0;

   private int rx1;

   private int x;

   private int y;

   private int hide;

   private int mobTime;

   private int team;

   public PlayerLifeData(int lifeId, String type, int cy, int f, int fh, int rx0, int rx1, int x, int y, int hide, int mobTime, int team) {
      this.lifeId = lifeId;
      this.type = type;
      this.cy = cy;
      this.f = f;
      this.fh = fh;
      this.rx0 = rx0;
      this.rx1 = rx1;
      this.x = x;
      this.y = y;
      this.hide = hide;
      this.mobTime = mobTime;
      this.team = team;
   }

   public int getLifeId() {
      return lifeId;
   }

   public String getType() {
      return type;
   }

   public int getCy() {
      return cy;
   }

   public int getF() {
      return f;
   }

   public int getFh() {
      return fh;
   }

   public int getRx0() {
      return rx0;
   }

   public int getRx1() {
      return rx1;
   }

   public int getX() {
      return x;
   }

   public int getY() {
      return y;
   }

   public int getHide() {
      return hide;
   }

   public int getMobTime() {
      return mobTime;
   }

   public int getTeam() {
      return team;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/PlayerLifeData.scala
package client.database.data

class PlayerLifeData(private var _lifeId: Int, private var _type: String, private var _cy: Int, private var _f: Int,
                     private var _fh: Int, private var _rx0: Int, private var _rx1: Int, private var _x: Int,
                     private var _y: Int, private var _hide: Int, private var _mobTime: Int, private var _team: Int) {
  def lifeId: Int = _lifeId

  def theType: String = _type

  def cy: Int = _cy

  def f: Int = _f

  def fh: Int = _fh

  def rx0: Int = _rx0

  def rx1: Int = _rx1

  def x: Int = _x

  def y: Int = _y

  def hide: Int = _hide

  def mobTime: Int = _mobTime

  def team: Int = _team
}
